code/__HELPERS/constants.dm

```diff
@@ -1,2 +1,2 @@
-var/global/const/TICKS_IN_DAY = 864000
-var/global/const/TICKS_IN_SECOND = 10
+var/const/TICKS_IN_DAY = 864000
```

Comment:
So you're basically going off the logic of a universe where DM would allow a secondary world object AND global variables are stored on it?